lms/envs/test.py

```diff
@@ -460,3 +460,8 @@
 FEATURES['ENABLE_COURSEWARE_SEARCH'] = True
 # Use MockSearchEngine as the search engine for test scenario
 SEARCH_ENGINE = "search.tests.mock_search_engine.MockSearchEngine"
+
+# Facebook app creds so jenkins tests succeed
+FACEBOOK_APP_SECRET = "6c26348ef355f53a531890e980ddc731"
```

Comment:
@DanielEidan Thanks for following through on this PR.
The only gating question I have is whether this `FACEBOOK_APP_SECRET` is a fake or real value?  If it's a real value, we cannot merge it into our codebase.